pkg/volume/iscsi/iscsi_util.go

```diff
@@ -205,6 +205,16 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 	iscsiTransport = extractTransportname(string(out))
 
 	bkpPortal := b.Portals
+
+	// create new iface and copy parameters pre-configured iface to the created iface
+	if b.InitiatorName != "" {
+		err = cloneIface(b, bkpPortal[0])
+		if err == nil {
+			// update iface name with <target portal>:<volume name>
+			b.Iface = bkpPortal[0] + ":" + b.volName
```

Comment:
Do you mean max len of iface which iscsiadm/OS is acceptable?

Here is an example of automatically created iface.
```
sudo iscsiadm -m iface
default tcp,<empty>,<empty>,<empty>,<empty>
iser iser,<empty>,<empty>,<empty>,<empty>
stable tcp,<empty>,<empty>,<empty>,<empty>
192.168.122.85:3260:pv0001 tcp,<empty>,<empty>,<empty>,iqn.1994-05.com.redhat:185ce16b55ad
```